packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/gleam/value.variable.scope

```diff
@@ -0,0 +1,21 @@
+let a = 3
+
+---
+
+[Content] = 0:8-0:9
+          >-<
+0| let a = 3
+
+[Removal] = 0:7-0:9
+         >--<
+0| let a = 3
```

Comment:
should this removal range include the equals?

Reply:
yes that's what we usually do, esp if declaring a variable without assigning a value is supported in the language